Remotion/ObjectBinding/Web.ClientScript/Scripts/BocAutoCompleteReferenceValue.UI.ts

```diff
@@ -1174,6 +1152,16 @@ namespace Remotion.BocAutoCompleteReferenceValue
             this.element.style.position = 'fixed';
             LayoutUtility.Hide(this.element);
 
+            this.politeAnnouncementElement = document.createElement("ul");
+            this.politeAnnouncementElement.ariaLive = "polite";
+            this.politeAnnouncementElement.classList.add("screenReaderText");
+            this.element.appendChild(this.politeAnnouncementElement);
+
+            this.assertiveAnnouncementElement = document.createElement("ul");
+            this.assertiveAnnouncementElement.ariaLive = "assertive";
+            this.assertiveAnnouncementElement.classList.add("screenReaderText");
+            this.element.appendChild(this.assertiveAnnouncementElement);
```

Comment:
https://www.w3.org/WAI/WCAG21/Techniques/aria/ARIA22

I'm wondering if instead of an UL, we shouldn't use a simple div with aria-role=status and update the text content of this DIV. As I understand it, for what we need, a UL isn't the best fit.

Reply:
I haven't tested `role=status` but I assume the reading behavior should mimic an attribute with the `aria-live="..."` set. The problem I faced was that text changes were not read correctly if the text changed to quickly. The list is a workaround to ensure that the message is announce again when a new message is added (even if that message is the same as the previous message).

Reply:
Regarding best fit: Since you can't browse this element it doesn't really matter what we pick here. `role=status` is probably best if it is a visual element that you can browse so that you are informed that this element might change in the future. As we just want the message and you can't browse to the element anyways, I think it does not matter what element we use for the announcement.

Reply:
Note for finishing the PR: we should use a DIV/SPAN instead of a UL, it just doesn't seem right. 
Note for finishing the PR: we already have a working example in `InformationPopUp` class that uses `aria-atomic` and `role=alert` and replaces the contents. adjust to this pattern to better symmetry of used patterns.

Reply:
Note on best fit: alert vs status since it's not an error, status might really be correct, regardless of it being visible to seeing users.